### PR TITLE
Issue #3045492 by mtodor: Backend performance testing

### PR DESCRIPTION
**Pull request by mtodor**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-3.x/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: tests/src/Nightwatch/Commands/select2/setValue.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**
+ * @file
+ * Sets tag for element. It can be existing tag or new one.
+ *
+ * This provides a custom command, .setSelect2Value()
```
Comment (alexpott) [inline tests/src/Nightwatch/Commands/select2/setValue.js]: select2.setValue?

File: tests/src/Nightwatch/Commands/select2/setValue.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ *   The element selector.
+ * @param {string} name
+ *   The tag name.
+ * @param {string} waitFor
+ *   The element selector that should be waited for.
+ *
+ * @return {object}
+ *   The 'browser' object.
+ */
+exports.command = function setSelect2Value(field, name, waitFor) {
```
Comment (alexpott) [inline tests/src/Nightwatch/Commands/select2/setValue.js]: I think this can be setValue

**Review by alexpott — COMMENTED** [review]
Just a small fix left. The command organisation looks great!

File: tests/src/Nightwatch/Commands/select2/setValue.js (hunk 2 above)
Comment (mtodor) [reply tests/src/Nightwatch/Commands/select2/setValue.js]: 👍

File: tests/src/Nightwatch/Commands/select2/setValue.js (hunk 1 above)
Comment (mtodor) [reply tests/src/Nightwatch/Commands/select2/setValue.js]: 👍

File: scripts/travis/06-test-source-code.sh [hunk 3]
```diff
@@ -10,6 +10,12 @@ phpcs --standard=DrupalPractice --extensions=php,module,inc,install,test,profile
 if ! [ -x "$(command -v eslint)" ]; then
     npm install -g eslint
 fi
+
+# Install ESLint requirements
+npm install -g eslint-config-airbnb eslint-plugin-jsx-a11y eslint-plugin-react eslint-plugin-import
+npm install -g prettier eslint-config-prettier eslint-plugin-prettier
```
Comment (chrfritsch) [inline scripts/travis/06-test-source-code.sh]: https://www.drupal.org/node/1955232 says we can just install  eslint-config-drupal  to get all required dependencies

**Review by alexpott — APPROVED** [review]
This looks great

**Pull request merged**